src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/sentencing/SentencingReconciliationService.kt

```diff
@@ -29,8 +30,36 @@ class SentencingReconciliationService(
         async { adjustmentsApiService.getAdjustments(prisonerId.offenderNo) }
     }.awaitBoth()
 
-    log.debug("will check adjustments for prisoner with ${prisonerId.offenderNo} booking: ${prisonerId.bookingId} : DPS has adjustments ${dpsAdjustments != null}")
-    return null
+    val nomisCounts: AdjustmentCounts = nomisAdjustments.let {
+      AdjustmentCounts(
+        lawfullyAtLarge = it.keyDateAdjustments.count { it.adjustmentType.code == "LAL" },
+        unlawfullyAtLarge = it.keyDateAdjustments.count { it.adjustmentType.code == "UAL" },
+        restorationOfAdditionalDaysAwarded = it.keyDateAdjustments.count { it.adjustmentType.code == "RADA" },
+        additionalDaysAwarded = it.keyDateAdjustments.count { it.adjustmentType.code == "ADA" },
+        specialRemission = it.keyDateAdjustments.count { it.adjustmentType.code == "SREM" },
+      )
+    }.let {
+      it.copy(
+        remand = nomisAdjustments.sentenceAdjustments.count { it.adjustmentType.code in listOf("RSR", "RX") },
+        unusedDeductions = nomisAdjustments.sentenceAdjustments.count { it.adjustmentType.code == "UR" },
+        taggedBail = nomisAdjustments.sentenceAdjustments.count { it.adjustmentType.code in listOf("S240A", "RST") },
```

Comment:
I wonder if these constant listOf()'s get optimised out to a more global scope 🤔

Reply:
Yes good point, I'll that in next PR